### PR TITLE
Updates Config availability in Parse Server

### PR DESCRIPTION
**Pull request by natanrolnik**

### CODE REVIEW

**Comment by drew-gross** [comment]
The config in parse-server is currently experimental, and may change before we enable it by default. Can you mention that in this PR?

File: en/server/migrating.mdown [hunk 1]
```diff
@@ -185,7 +185,7 @@ This is always allowed in Parse Server.
 
 **Config**
 
-This is not supported. You can create config variables in Node that can be changed and deployed. Or, it would be relatively straightforward to create a Node version of Parse Config with a dashboard to change these variables without a deployment.
+While the Parse hosted Config can be changed via dashboard, with Parse Server an initial version of Parse Config is available via REST API. To add or update fields, you can call `/config` with `POST` with the desired keys and values, and to get the whole Config, just call a `GET` on the same `/config` endpoint of your app. Updating Config with Parse Server requires using the Master Key; otherwise, it will fail. For more information, check [the Pull Request that added it.](https://github.com/ParsePlatform/parse-server/pull/283)
```
Comment (guich-wo) [inline en/server/migrating.mdown]: PUT, not POST

**Comment by guich-wo** [comment]
Not experimental any more.
HTTP method was fixed since the PR was created, now using PUT instead of POST for config update

**Comment by drew-gross** [comment]
Also you can actually change it from your own Parse Dashboard now.

**Pull request closed without merging**